ocean-client/src/programs/vault-maxi-program.ts

```diff
@@ -46,12 +46,14 @@ export class VaultMaxiProgram extends CommonProgram {
     
     private readonly targetCollateral: number
     private readonly lmPair: string
+    private readonly keepWalletClean: boolean
 
     constructor(store: Store, walletSetup: WalletSetup) {
         super(store, walletSetup);
 
         this.lmPair = this.settings.LMToken + "-DUSD"
         this.targetCollateral = (this.settings.minCollateralRatio + this.settings.maxCollateralRatio) / 200
+        this.keepWalletClean = process.env.VAULTMAXI_KEEP_CLEAN !== "false" ?? true
```

Comment:
Do we want to make all "more tech savvy or pro" functionalities via environment variables? To avoid someone accidentally activating those?

Reply:
yes we do :D